README.md

```diff
@@ -0,0 +1,767 @@
+
+# Abstract
+
+This specification describes the semantics and serialization format for [UCAN] delegation between principals. UCAN Delegation provides a cryptographically verifiable container, batched capabilities, hierarchical authority, and extensible conditions.
+  
+# Introduction
+
+UCAN Delegation is a certificate capability system with runtime-extensibility, ad hoc conditions, cacheability, and focused on ease of use and interoperability. Delegations act as a proofs for [UCAN Invocation]s.
+
+Delegation provides a way to "transfer authority without transferring cryptographic keys". As an authorization system, it is more interested in "what can be done?" than a list of "who can do what?". For more on how Delegation fits into UCAN, please refer to the [high level spec][UCAN].
+
+# Delegation (Envelope)
+
+Delegations MUST include a signature that validates against the `iss` DID. A Delegation Payload on its own MUST NOT be considered a valid Delegation.
+
+``` mermaid
+flowchart TD
+    subgraph Delegation
+        SignatureBytes["Signature (raw bytes)"]
+      
+        subgraph SigPayload ["Signature Payload"]
+            VarsigHeader["Varsig Header"]
+
+            subgraph DelegationPayload ["Delegation Payload"]
+                iss
+                sub
+                can
+                args
+                prf
+                etc["..."]
+            end
+        end
+    end
+```
+
+``` js
+{
+  "s": {"/": {"bytes": "7aEDQLYvb3lygk9yvAbk0OZD0q+iF9c3+wpZC4YlFThkiNShcVriobPFr/wl3akjM18VvIv/Zw2LtA4uUmB5m8PWEAU"}}
+  "p": {
+    "h": {"/": {"bytes": "NBIFEgEAcQ"}},
+    "ucan/d/1.0.0-rc.1": {
+      "iss": "did:plc:ewvi7nxzyoun6zhxrhs64oiz",
+      "sub": "did:key:z6MkrZ1r5XBFZjBU34qyD8fueMbMRkKw17BZaq2ivKFjnz2z",
+      "can": "crud/create",
+      "args": {
+        "uri": "https://example.com/blog/posts",
+        "headers": {
+          "content-type": "application/json"
+        },
+        "payload": {
+          "title": "UCAN for Fun an Profit",
+          "body": "UCAN is great!",
+          "topics": ["authz", "journal"],
+          "draft": true
+        }
+      },
+      "nonce": {"/": {"bytes": "TWFueSBopvcs"}},
+      "exp": 1697409438
+    }
+  }
+}
+```
+
+| Field | Type               | Required | Description                                           |
+|-------|--------------------|----------|-------------------------------------------------------|
+| `s`   | `Signature`        | Yes      | A signature by the Payload's `iss` over the `p` field |
+| `p`   | `SignaturePayload` | Yes      | The content that was signed                           |
+
+## Signature Payload
+
+| Field               | Type                | Required | Description              |
+|---------------------|---------------------|----------|--------------------------|
+| `h`                 | `VarsigHeader`      | Yes      | The Varsig header        |
```

Comment:
I'm still not sure what goes here, I don't think it was covered anywhere in the spec